functions/constants.js

```diff
@@ -1,160 +1,4 @@
-	'sudoku-solution-validator',
-	'arithmetic-expressions-evaluator',
-	'a-simple-tic-tac-toe-class',
-	'connect-four-who-won',
-	'chinese-numeral-encoder',
-	'parseint-reloaded',
-	'sum-of-intervals',
-	'remove-zeros',
-	'how-low-can-you-go',
-	'default-arguments',
-	'matrix-determinant',
-	'algebraic-lists',
-	'word-char-and-line-count',
-	'advanced-events',
-	'design-a-simple-automaton-finite-state-machine',
-	'reverse-polish-notation-calculator',
-	'hamming-numbers',
-	'dependency-injection',
-	'instant-runoff-voting',
-	'determining-if-a-graph-has-a-solution',
-	'explosive-sum',
-	'counting-change-combinations',
-	'es5-generators-i',
-	'lru-cache',
-	'undo-slash-redo',
-	'a-simplistic-tcp-finite-state-machine-fsm',
-	'float-to-binary-conversion',
-	'monads-the-list-monad',
-	'simple-memory-manager',
-	'decimal-to-any-rational-or-irrational-base-converter',
-	'algebraic-data-types',
-	'hello-w-dot-dot-dot-wait-what',
-	'pyramid-slide-down',
-	'binary-multiple-of-3',
-	'fancy-schmancy-sort',
-	'polydivisible-numbers',
-	'factorial-tail',
-	'set-the-card-game',
-	'total-increasing-or-decreasing-numbers-up-to-a-power-of-10',
-	'hash-hacker',
-	'largest-numeric-palindrome',
-	'fluent-calculator-1',
-	'simple-image-processing-part-i-edges-feature-growing-and-shrinking',
-	'better-add-floats',
-	'catching-car-mileage-numbers',
-	'pascals-triangle',
-	'sum-strings-as-numbers',
-	'vigenere-cipher-helper',
-	'skrzat',
-	'next-bigger-number-with-the-same-digits',
-	'large-factorials',
-	'infix-to-postfix-converter',
-	'born-to-be-chained',
-	'coordinates-validator',
-	'simplifying-multilinear-polynomials',
-	'decode-the-morse-code-advanced',
-	'next-smaller-number-with-the-same-digits',
-	'snail',
-	'text-align-justify',
-	'take-it-or-leave-it-interview-strategy',
-	'vending-machine',
-	'abc-esoteric-programming-number-2',
-	'esolang-interpreters-number-2-custom-smallfuck-interpreter',
-	'base-conversion',
-	'roman-numerals-decoder',
-	'pick-peaks',
-	'functional-streams',
-	'roman-numerals-encoder',
-	'unflatten-a-list-harder-than-easy',
-	'strings-mix',
-	'ranking-poker-hands',
-	'recover-a-secret-string-from-random-triplets',
-	'sum-by-factors',
-	'breadcrumb-generator',
-	'ten-pin-bowling',
-	'square-into-squares-protect-trees',
-	'longest-common-subsequence',
-	'magnet-particules-in-boxes',
-	'how-many-numbers-iii',
-	'guess-the-digits-and-expression',
-	't-dot-t-t-number-2-equal-to-24',
-	'helpers-for-a-3dgame-i-biggest-triangle-in-a-sphere',
-	'shortest-code-virus-in-apple',
-	'roboscript-number-3-implement-the-rs2-specification',
-	'twice-linear',
-	'validate-sudoku-with-size-nxn',
-	'simple-fun-number-27-rectangel-rotation',
-	'getting-along-with-integer-partitions',
-	'create-a-funnel',
-	'valid-braces,ip-validation',
-];
+exports.dailyccChannel = '774024031380439091';
+exports.affirmationsChannel = '773420650744250408';
+exports.blaccSmithLogo =
+	'https://avatars0.githubusercontent.com/u/73925140?s=200&v=4';
```

Comment:
If we have the structure of **functions/dailyCC/constants.js** for the other file you edited can we place this file like **functions/global/constants.js** for consistency purposes

Reply:
there's already a top level constants file. The folders inside **functions/** are meant for bot features